packages/federation/src/supergraph.ts

```diff
@@ -770,6 +789,9 @@ export function getSubschemasFromSupergraphSdl({
             break;
           }
         }
+        if (isOrphan) {
+          (interfaceInSubgraph as any).kind = Kind.OBJECT_TYPE_DEFINITION;
```

Comment:
Please avoid `any`s as much as possible. Rather add a bunch of if statements to satisfy TS.

Reply:
`kind` is readonly so I couldn't find another way :(

Reply:
Then do this instead:
```suggestion
          // @ts-expect-error I write even though readonly because <explain>
          interfaceInSubgraph.kind = Kind.OBJECT_TYPE_DEFINITION;
```

Reply:
Good idea :D